MiniAn/minian_run.py

```diff
@@ -12,51 +12,15 @@
 sys.path.append('..')
 import utilities as utils
 import minian_utilities as mn_utils
-
-# Import for MiniAn lib
-from minian.utilities import TaskAnnotation, get_optimal_chk, custom_arr_optimize, save_minian, open_minian
-from minian.preprocessing import denoise, remove_background
-from minian.initialization import seeds_init, pnr_refine, ks_refine, seeds_merge, initA, initC
-from minian.cnmf import compute_trace, get_noise_fft, update_spatial, update_temporal, unit_merge, update_background, compute_AtC
-from minian.motion_correction import apply_transform, estimate_motion
+import minian_parameter as mn_param
+import minian_main      as mn_main
+import minian_preview   as mn_preview
+import minian_text_def  as mn_txt
 
 # Import for PyInstaller
 from multiprocessing import freeze_support
 freeze_support()
 
-# Text definitions
-START_CLUSTER       = "Starting cluster..."
-ADVANCED_BAD_TYPE   = "One of the advanced settings is not of a python type"
-LOAD_DATA           = "Loading dataset to MiniAn..."
-ONE_PARM_WRONG_TYPE = "One parameter of {0} function is of the wrong type"
-NO_CELLS_FOUND      = "No cells where found"
-PREPROCESS          = "Pre-processing..."
-PREPROC_REMOVE_GLOW = "Pre-processing: removing glow..."
-PREPROC_DENOISING   = "Pre-processing: denoising..."
-PREPROC_REMOV_BACKG = "Pre-processing: removing background..."
-PREPROC_SAVE        = "Pre-processing: saving..."
-CORRECT_MOTION_ESTIM_SHIFT  = "Correcting motion: estimating shifts..."
-CORRECT_MOTION_APPLY_SHIFT  = "Correcting motion: applying shifts..."
-PREP_DATA_INIT              = "Preparing data for initialization..."
-INIT_SEEDS                  = "Initializing seeds..."
-INIT_SEEDS_PNR_REFI         = "Initializing seeds: PNR refinement..."
-INIT_SEEDS_KOLSM_REF        = "Initializing seeds: Kolmogorov-Smirnov refinement..."
-INIT_SEEDS_MERG             = "Initializing seeds: merging..."
-INIT_COMP                   = "Initializing components..."
-INIT_COMP_SPATIAL           = "Initializing components: spatial..."
-INIT_COMP_TEMP              = "Initializing components: temporal..."
-INIT_COMP_MERG              = "Initializing components: merging..."
-INIT_COMP_BACKG             = "Initializing components: background..."
-RUN_CNMF_ITT                = "Running CNMF {0} itteration: "
-RUN_CNMF_ESTIM_NOISE        = RUN_CNMF_ITT + "estimating noise..."
-RUN_CNMF_UPDAT_SPATIAL      = RUN_CNMF_ITT + "updating spatial components..."
-RUN_CNMF_UPDAT_BACKG        = RUN_CNMF_ITT + "updating background components..."
-RUN_CNMF_UPDAT_TEMP         = RUN_CNMF_ITT + "updating temporal components..."
-RUN_CNMF_MERG_COMP          = RUN_CNMF_ITT + "merging components..."
-RUN_CNMF_SAVE_INTERMED      = RUN_CNMF_ITT + "saving intermediate results..."
-SAVING_FINAL                = "Saving final results..."
-SAVING_TO_DORIC             = "Saving data to doric file..."
-
 kwargs = {}
```

Comment:
I'm confused -- what are `kwargs`, `params_doric`, `danse_parameters`, `minian_params`? Do you think they can be renamed to give more clarity?